definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -105,5 +105,19 @@
     "CaseEventID": "fl401ManageFlags",
     "UserRole": "caseworker-privatelaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "citizen-case-submit-with-hwf",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "citizen-case-submit-with-hwf",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "CRUD": "R"
```

Comment:
can we do it with access control. we will have reduced number of code. also authorisation should be granted to judges, la, system user, system update in order to view it in history.

Reply:
Done